minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloConfig.java

```diff
@@ -108,6 +108,19 @@ public MiniAccumuloConfig setZooKeeperStartupTime(long zooKeeperStartupTime) {
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfig setExistingZooKeepers(String existingZooKeepers) {
```

Comment:
I think it would be the following:
* Getter should return `null` if not set
* Clear it by passing `null` to setter. Setter should probably disallow setting an empty string.
* `setZooKeeperPort` or `setExistingZooKeepers` should be mutually exclusive